mart/configs/callbacks/no_grad_mode.yaml

```diff
@@ -0,0 +1,2 @@
+attack_in_eval_mode:
```

Comment:
Shall we make an accurate name for this?

Reply:
This should go away in #141.